lib/matplotlib/tests/test_colors.py

```diff
@@ -848,6 +848,36 @@ def test_to_rgba_array_single_str():
                              "sequence"):
         mcolors.to_rgba_array("rgbx")
 
+@pytest.mark.parametrize("color,factor", [
+    ([1, 0, 0], 1),
+    ([1, 0, 0], 1.1),
+    ([1, 1, 1], 3),
+    ([0.4, 0.3, 0.1], 1.04),
+    ([0.1, 0.1, 0.1], 1.03),
```

Comment:
I don’t see that any of these explicitly test the saturation wrapping when value exceeds 1